layouts/page/superpowers.html

```diff
@@ -0,0 +1,276 @@
+                    </div>
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0">
+                            <a href="{{ relref . "/docs/intro/concepts" }}">
+                                <span class="align-middle text-blue-700 transition-all hover:mr-1">Architecture</span>
+                                <span class="text-blue-400 text-lg">&rarr;</span>
+                            </a>
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 leading-relaxed">
+                            Stop reinventing the wheel with each new microservice component and create
+                            self-service infrastructure for your teams. Leverage an ecosystem of
+                            libraries that implement best practices.
+                        </p>
+                    </div>
+                </li>
+                <li class="md:w-1/2 flex items-start">
+                    <div class="bg-blue-100 border-4 border-blue-200 rounded-full p-6" style="fill: #4387c7;">
+                        <svg class="h-12 w-12" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><defs><style>.fa-secondary{opacity:.4}</style></defs><path d="M466.5 83.67l-192-80a48.15 48.15 0 0 0-36.9 0l-192 80A48 48 0 0 0 16 128c0 198.5 114.5 335.69 221.5 380.29a48.15 48.15 0 0 0 36.9 0C360.1 472.58 496 349.27 496 128a48 48 0 0 0-29.5-44.33zm-47.2 114.21l-184 184a16.06 16.06 0 0 1-22.6 0l-104-104a16.07 16.07 0 0 1 0-22.61l22.6-22.6a16.07 16.07 0 0 1 22.6 0l70.1 70.1 150.1-150.1a16.07 16.07 0 0 1 22.6 0l22.6 22.6a15.89 15.89 0 0 1 0 22.61z" class="fa-secondary"/><path d="M419.3 197.88l-184 184a16.06 16.06 0 0 1-22.6 0l-104-104a16.07 16.07 0 0 1 0-22.61l22.6-22.6a16.07 16.07 0 0 1 22.6 0l70.1 70.1 150.1-150.1a16.07 16.07 0 0 1 22.6 0l22.6 22.6a15.89 15.89 0 0 1 0 22.61z" class="fa-primary"/></svg>
+                    </div>
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0">
+                            <a href="{{ relref . "docs/guides/crossguard" }}">
+                                <span class="align-middle text-blue-700 transition-all hover:mr-1">Policy</span>
+                                <span class="text-blue-400 text-lg">&rarr;</span>
+                            </a>
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 md:border-b-0 leading-relaxed">
+                            Enforce standards for deployments with auditing capabilities across roles,
```

Comment:
Enforce security, compliance, and cost controls to prevent mistakes from reaching production. Define custom policies using your favorite language and flexibly apply them across your projects.

Reply:
looks fine - will need to see in preview if the extra text causes issues.